app/views/repositories/_form.html.erb

```diff
@@ -94,3 +108,11 @@
   <%= f.submit t('save'), class: 'btn btn-primary' %>
   <%= link_to t('back'),  project_path(@project_id), class: 'btn btn-default' %>
 </div>
+
+<script type="text/javascript">
+  var _repository_branch = new Repository.Branch;
```

Comment:
I know why the var name starts with underscore. But does someone else know the reason?